test cases/common/58 run target/fakeburner.py

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
+
+from __future__ import print_function
 
```

Comment:
It will be run with Python 3 now. In the future someone may change this project or the way the code works or anything, really. This is about reliability, which comes from defense in depth. I don't care what it is run with, only that it works and I never have to debug Python versions again.